src/Mod/BIM/BimStatus.py

```diff
@@ -121,6 +121,7 @@ def toggleSaveSettings(action):
             else:
                 statuswidget = FreeCADGui.UiLoader().createWidget("Gui::ToolBar")
                 statuswidget.setObjectName("BIMStatusWidget")
+                statuswidget.setWindowTitle(translate("A context menu action used to show or hide this toolbar widget", "BIMStatusWidget"))
```

Comment:
```suggestion
                statuswidget.setWindowTitle(translate("A context menu action used to show or hide this toolbar widget", "BIM status widget"))
```

Same comment as before.

Reply:
This call is backwards: the context comes first, the string to translate second. I presume that the "BimStatusWidget" is actually the context here, so it's correct as written, just in the wrong place.

Reply:
@chennes, I wrote according to this:


https://github.com/FreeCAD/FreeCAD/blob/d5a1098add95e829ef8396736d23b043fbf9fa83/src/Mod/Draft/draftutils/translate.py#L46-L50

Reply:
Updated suggestion accordingly.

Reply:
See also https://github.com/FreeCAD/DevelopersHandbook/blob/main/technical/translation.md